### PR TITLE
javascript-javascript3-week2

### PR DESCRIPTION
**Pull request by SvitlanaBurlaka**

### CODE REVIEW

File: javascript/javascript3/week2/index.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    .then(() => fetch("https://www.swapi.tech/api/people/"))
+    .then((response) => response.json())
+    .then((data) => console.log(data));
+
+(async function() {
+    await setTimeoutPromise(3000);
+    const response = await fetch("https://www.swapi.tech/api/people/");
+    const data = await response.json();
+    console.log(data);
+})();
```
Comment (ElhamDargi) [inline javascript/javascript3/week2/index.js:95]: Great job! 😊 
You did the homework in the best way.

File: javascript/javascript3/week2/Visual Promise/main.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+// translateOneByOne();
+
+function translateAllAtOnce() {
+    Promise.all([
+        moveElement(redBox, { x: 20, y: 300 }),
+        moveElement(blueBox, { x: 400, y: 300 }),
+        moveElement(greenBox, { x: 400, y: 20 }),
+    ]).then(() => console.log("Boxes are moved"));
+}
+translateAllAtOnce();
```
Comment (ElhamDargi) [inline javascript/javascript3/week2/Visual Promise/main.js:27]: Excellent!!!!

File: javascript/javascript3/week2/index.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
+// Movies exercise
+const badMovies = [];
+const badMovies2000 = [];
+
+fetch(
+        "https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json"
+    )
+    .then((response) => response.json())
+    .then((data) => {
+        data.forEach((element) => {
```
Comment (ElhamDargi) [inline javascript/javascript3/week2/index.js:10]: You can also use the "filter" to find the bad movies.

        const badMovies= data.filter((movie) => movie.rating <5);
        console.log(badMovies);
        const badMovies2000= badMovies.filter((movie) => movie.year > 2000);
        console.log(badMovies2000);

Comment (SvitlanaBurlaka) [reply javascript/javascript3/week2/index.js:10]: Thank you so much for reviewing my homework!

**Pull request merged**